unratified/rv_zvknh

```diff
@@ -0,0 +1,12 @@
+# Zvknh - Vector Crypto SHA-256/SHA-512 (Hash)
```

Comment:
The spec diffrentiates between zvknha and zvknhb - indicating there is not extension named zvknh. In that case these instructions must be split into multiple files - rv_zvknha and rv_zvknhb

Reply:
Done.

Reply:
For some reason Github keeps showing this as a needed change, so I unresolved and will mark revolved again. Let's see if this does the trick.

Reply:
Yes, even if you request review again, until someone who has requested change(s) does not approve the PR it will appear as needing change.